components/PageNavigation.vue

```diff
@@ -20,6 +20,31 @@
         </span>
       </SmartLink>
     </li>
+    <!-- sso links -->
+    <template v-if="enableAuthLinks">
+      <li
+        v-if="!isAuthenticated"
+        class="nav-item"
+      >
+        <b-link
+          class="nav-link"
+          @click="login()"
+        >
+          <span>{{ $t('account.linkLogin') }}</span>
+        </b-link>
+      </li>
+      <li
+        v-else
+        class="nav-item"
+      >
+        <b-link
+          href="/account/profile"
```

Comment:
This does not utilise the Vue router, resulting in an SSR always being made.

Instead, use: `:to="localePath({ name: 'account-profile' })"`